pkg/common/common.go

```diff
@@ -29,7 +29,8 @@ const (
 	DiskImageName = "disk.img"
 	// ImporterWritePath provides a constant for the cmd/cdi-importer/importer.go executable
 	ImporterWritePath = ImporterVolumePath + "/" + DiskImageName
-
+	// ImporterWriteBlockPath provides a constant for the path where the PV is mounted.
+	ImporterWriteBlockPath = "/dev/import"
```

Comment:
I am going to re-iterate Freds concern here. I don't think the block device is a constant in the container. AFAIK it will show up as whatever device is on the host. Try creating a /dev/loop0 or something not called /dev/import for a block device and running this code, I get a feeling it will fail.

Reply:
With the latest code, I create /dev/import block device on the node (dd import, mknod /dev/import, losetup /dev/import import). The device volume in the pod spec refer to /dev/import as well. Will that work?

Reply:
Right, the point it, we can't guarantee that the block device name is something constant. Depending on the manufacturer of the block device (or a loop back block device we use for testing) the name of the device will be different. So we can't assume inside the pod that the device path will be something constant, the controller has to parse it out of the PV and pass it into the pod.

Reply:
Alright, after some more research, you are right this can be hard coded, you define the path the device is mapped to in the pod spec, so it doesn't matter what the underlying host device name is, that is defined in the PV spec which then is connected to the PVC, and you simply map the device associated with the PVC to the container with whatever name you want. So this is fine.

Reply:
so apparently, it seems eventually, that you were right and not me.
The path in the PV let's say is /dev/import1.
The path in the pod that the device is mapped to is /dev/import.
On the hosts I created /dev/import, /dev/import1 and /dev/import2 with loopbackfile, loopbackfil1 and loopbackfile2 accordingly.
it ended up importing the image on the host to loopbackfile2. I was expecting it to be imported to loopbackfile1 (as the path in the PV is /dev/import1). 
Still testing....